index.html

```diff
@@ -39,131 +39,158 @@ <h3 class="cover-links">LET'S CONNECT</h3>
+            <li><a href="#" class="fa fa-google"></a></li>
+            <li><a href="#" class="fa fa-github"></a></li>
+            <li><a href="#" class="fa fa-javascript"></a></li>
+          </ul>
+          <button class="resume">Get My Resume</button>
+      </div>
+      <div class="about-myself-languages">
+          <h3 class="languages">Languages</h3>
+          <ul class="framework-skills">
+            <li><img src="images/skills/html.svg" alt="HTML skill" /></li>
+            <li><img src="images/skills/css.svg" alt="CSS skill" /></li>
+            <li><img src="images/skills/js.svg" alt="Javascript skill" /></li>
+          </ul>
+          <ul class="frameworks" style="list-style: none;">
+            <li class="framework1">Frameworks</li>
+            <li class="greater-tag">></li>
+          </ul>
+          <hr>
+          <ul class="frameworks" style="list-style: none;">
+            <li class="framework1">Skills</li>
+            <li class="greater-tag">></li>
+          </ul>
+          <hr>
       </div>
-        <p class="highlights">
-          A daily selection of privately personalized reads no accounts for
-          sign-ups required.
+      <!-- Contact page -->
+      <div id="my-contact" class="contact">
+        <div class="footer-content">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-content">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
         </p>
-        <button class="tag1">HTML</button>
-        <button class="tag2">CSS</button>
-        <button class="tag3">Javascript</button>
-      
-        <button class="project-detail">See Project</button>
-    </div>
-  </section>
-
-    <!-- Third project -->
-  <section class="work-page-right">
-   <div class="card3-image">
-    <img src="images/works/tonic/cover1.svg" alt="Poster for tonic app" style="width: 100%;" />
-   </div>
-    <div class="card3-content">
-      <h3 class="work-title">Multy-Post Stories</h3><br><br>
-      <div class="clients">
-        <h4>CANOPY</h4>
-        <img class="img-clients" src="images/works/Counter.svg" alt="">
-        <h4 style="color: #6b778c">Back End Dev</h4>
-        <img class="img-clients" src="images/works/Counter.svg" alt="">
-        <h4 style="color: #6b778c">2015</h4>
+        <span id="error"></span>
+        <form action="https://formspree.io/f/xgeqpzkb" method="POST" class="my-form" name="myForm">
+          <input maxlength="30" type="text" name="name" placeholder="Full name" class="inputField" required>
+          <input id= "email" type="email" name="email" placeholder="Email" class="inputField" required>
+          <textarea maxlength="500" name="text" class="textareaField" cols="30" rows="10" placeholder="Write your message here" required></textarea>
+          <button type="submit" class="my-button">Get It Touch</button>
```

Comment:
- [ ] Good job adding your form 👍🏽. Kindly ensure that the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

![image](https://user-images.githubusercontent.com/63473584/212973901-143625dd-bb04-4573-bf27-38e9eadf49f5.png)